website_canonical_url/tests/test_rendering.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you avoid renaming this file? Thanks

Reply:
`test_rendering` sounded more clear then `test_tags` but I don't mind that much :wink: